docs/installation/install.md

```diff
@@ -26,16 +26,10 @@ Hence, make sure you selected the correct target cluster before issuing *liqoctl
 
 **Supported CNIs**
 
-Liqo supports Kubernetes clusters using the following CNIs: [Flannel](https://github.com/flannel-io/flannel), [Calico](https://www.tigera.io/project-calico/), [Canal](https://github.com/projectcalico/canal), [Weave](https://github.com/weaveworks/weave).
-Additionally, partial support is provided for [Cilium](https://cilium.io/), although with the limitations listed below.
+Liqo supports Kubernetes clusters using the following CNIs: [Cilium](https://cilium.io/), [Flannel](https://github.com/flannel-io/flannel), [Calico](https://www.tigera.io/project-calico/), [Canal](https://github.com/projectcalico/canal), [Weave](https://github.com/weaveworks/weave).
```

Comment:
@cheina97 It seems to me that Canal can be removed, as it does not represent a very active project.
Am I wrong?

Reply:
https://docs.tigera.io/calico/latest/getting-started/kubernetes/flannel/install-for-flannel

The project seems still alive, but the link provided in the old canal repository is dead.
We can change the link in our documentation with this one